pkg/util/util.go

```diff
@@ -29,3 +30,10 @@ func GetVolumeNameFromEngineName(engineName string) string {
 func BytesToMiB(bytes uint64) uint64 {
 	return bytes / 1024 / 1024
 }
+
+func RemovePrefix(path, prefix string) string {
+	if strings.HasPrefix(path, prefix) {
+		return strings.TrimPrefix(path, prefix)
+	}
+	return path
```

Comment:
Can just `return strings.TrimPrefix(path, prefix)`

Reply:
NIT: is this function necessary...? Why not directly call `strings.HasPrefix`...?

Reply:
Not necessary. Can do `diskPath := strings.TrimPrefix(path, prefix)` directly